scripts/deploy.sh

```diff
@@ -16,9 +16,9 @@
 #
 
 REMOTE_APP_DIR="/var/www/telemetry"
-DEBIAN_PKGS="build-essential python3 python3-dev python3-pip python3-virtualenv libpq-dev nginx git"
-REDHAT_PKGS="gcc gcc-c++ make python34 python34-devel python34-pip python34-virtualenv postgresql-devel postgresql-server postgresql-contrib nginx git policycoreutils-python"
-CLR_BUNDLES="application-server database-basic database-basic-dev python-basic os-clr-on-clr os-core-dev web-server-basic"
+DEBIAN_PKGS="build-essential python3 python3-dev python3-pip virtualenv libpq-dev nginx git uwsgi uwsgi-plugin-python3"
+REDHAT_PKGS="gcc gcc-c++ make python34 python34-devel python34-pip python34-virtualenv postgresql-devel postgresql-server postgresql-contrib nginx git policycoreutils-python uwsgi uwsgi-plugin-python3"
+CLR_BUNDLES="application-server database-basic database-basic-dev python-basic os-core-dev web-server-basic"
```

Comment:
Is os-clr-on-clr no longer needed? I don't see this called out in the commit message...